Microsoft.Toolkit.Uwp.SampleApp/SamplePages/OneDrive Service/OneDrivePage.xaml.cs

```diff
@@ -148,14 +143,7 @@ public OneDrivePage()
 
         private async void LogOutButton_Click(object sender, RoutedEventArgs e)
         {
-            if (_indexProvider == 3)
-            {
-                await Services.OneDrive.OneDriveService.GraphInstance.LogoutAsync();
-            }
-            else
-            {
-                await Services.OneDrive.OneDriveService.Instance.LogoutAsync();
-            }
+            await OneDriveService.Instance.LogoutAsync();
```

Comment:
I logged in with my MSA (using a clientId for MSA). I was able to see my files. I then logged out and changed the clientId to AAD clientID and logged in with my work account. The files didn't load. Closing the app and logging in with my work account from the beginning worked as expected and I could see my files.

Logout in between seems to not work as expected.

Reply:
Thanks - There appears to be a bug where the service maintains the client Id after sign-out so that when you switch it doesn't use the new one. I'll fix that.

Reply:
After investigation the bug was simply not resetting the visibility on the ListView